lxd/storage/drivers/driver_zfs.go

```diff
@@ -744,8 +744,8 @@ func (d *zfs) patchDropBlockVolumeFilesystemExtension() error {
 	return nil
 }
 
-// roundVolumeBlockSizeBytes returns size rounded to the nearest multiple of
-// zfs.blocksize for `vol` that is equal to or larger than sizeBytes.
+// roundVolumeBlockSizeBytes returns sizeBytes rounded up to the next multiple
+// of `vol`'s "zfs.blocksize".
```

Comment:
This comment change wrt to zfs.blocksize should be part of the `lxd/storage/drivers/zfs: Round to zfs.blocksize or 16KiB ` commit right?

Reply:
That's not unreasonable. Fixed.